plugins/reolink/src/reolink-api.ts

```diff
@@ -294,7 +294,7 @@ export class ReolinkCameraClient {
             op += 'Up';
 
         if (op) {
-            await this.ptzOp(op, Math.round(Math.abs(command?.pan || command?.tilt || 1) * 10));
```

Comment:
I used ceil instead of round here because the commands from the app where < 0.5 and they were rounded to 0 instead of 1